### PR TITLE
[GCI-riskycase] AC-639: Set Code Style

### PR DESCRIPTION
**Pull request by riskycase**
## Description of what I changed

Updated .gitignore and readme, and added a codestyle XML to .idea/codeStyles

## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-639

## Checklist: I completed these to help reviewers :)

- [x] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [x] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,6 +81,12 @@ Also, before creating a pull request, please run code review tools (Lint) and al
 ### Code Style
 The coding conventions used by OpenMRS are outlined [here](https://wiki.openmrs.org/display/docs/Coding+Conventions).
 
+Please follow the below instructions while contributing to the development of the app
```
Comment (f4ww4z) [inline README.md]:
```suggestion
Please follow the below instructions while contributing:
```

File: README.md [hunk 2]
```diff
@@ -81,6 +81,12 @@ Also, before creating a pull request, please run code review tools (Lint) and al
...
+1. When project is opened with Android Studio, go to File > Settings > Editor > Code Style
+2. Select Project from the Scheme dropdown
```
Comment (f4ww4z) [inline README.md]: You should be bolding options that user can pick, **like so**

File: README.md [hunk 3]
```diff
@@ -81,6 +81,12 @@ Also, before creating a pull request, please run code review tools (Lint) and al
 ### Code Style
 The coding conventions used by OpenMRS are outlined [here](https://wiki.openmrs.org/display/docs/Coding+Conventions).
```
Comment (f4ww4z) [inline README.md]: Reword this to point to https://wiki.openmrs.org/display/docs/Developer+How-To+Setup+And+Use+IntelliJ#DeveloperHow-ToSetupAndUseIntelliJ-SetupCodeStyleAndFormatForOpenMRS

File: README.md [hunk 4]
```diff
@@ -81,6 +81,12 @@ Also, before creating a pull request, please run code review tools (Lint) and al
...
 
+Please follow the below instructions while contributing to the development of the app
+1. When project is opened with Android Studio, go to File > Settings > Editor > Code Style
+2. Select Project from the Scheme dropdown
+3. Make sure that hard wrap is the same under Wrapping and Bases in Java and Other in XML language settings (currently 125)
```
Comment (f4ww4z) [inline README.md]: This statement is unclear. Specify the hard wrap specifically.
```suggestion
3. Make sure that **hard wrap** is set to **125** for Java and XML language settings.
```

**Comment by f4ww4z** [comment]
@riskycase did you also follow our PR tips and claim the issue in JIRA?

**Comment by riskycase** [comment]
Should've been more careful while rebasing, ended up deleting my own commit XD. Reopening PR
